docker/application.conf

```diff
@@ -35,3 +35,17 @@ openseedbox.zip.path=/rdr
 openseedbox.zip.manifestonly=false
 openseedbox.context.path=${OPENSEEDBOX_CONTEXT_PATH}
 openseedbox.assets.prefix=${OPENSEEDBOX_CONTEXT_PATH}/public
+
+@include.runtime=application.runtime.conf
+
+# Test overrides
+%test.db=mem
+%test.db.url=jdbc:h2:mem:testdb
```

Comment:
I'm testing against in-memory h2 database instead of PostgreSQL. :upside_down_face: 
It's easier than firing up a postgres container. :sweat_smile: